app/src/main/java/grinnell/appdev/edu/lyles/Adapters/BeerAdapter.java

```diff
@@ -19,6 +23,8 @@
  * adapter contains image, and description of catelog item
  */
 public class BeerAdapter extends ArrayAdapter<BeerItem> {
+    private static final String TAG = ArrayAdapter.class.getSimpleName();
```

Comment:
Using a tag is good in logging and testing things on your own. Unless it is particularly necessary, you shouldn't have a tag here. I like that you made it `static final`. Immediately, though, you don't need it :)